### PR TITLE
Including added_at response variable for found queries

### LINKED ISSUE #46: Append timestamp when querying a list
When an entry is found after querying a list, the integration should append the `timestamp` variable (or maybe it shoudl be called `added_at` or even something else?). This variable should contain the most recent timestamp found in the list of `entries` returned. The timestamp response variable should have type `time`.

The purpose of this variable is to allow the user to create rules based on when the entry was added to the list.

### PR DESCRIPTION
**Pull request by bhicks**
This change adds an `added_at` response variable for found queries. It returns as a `Time` type, so that time comparisons can be made from following steps.

Fixes #46 

@cgrayson can you take a look when you get a chance?

### CODE REVIEW

File: src/helper.coffee [hunk 1]
```diff
@@ -62,14 +62,6 @@ parseResponse = (res) ->
...
   switch process.env.NODE_ENV
     when 'production', 'test' then 'https://app.suppressionlist.com'
-    when 'staging' then 'https://staging.suppressionlist.com'
-    when 'development' then 'http://suppressionlist.dev'
-
-
-
-getBaseUrl = ->
-  switch process.env.NODE_ENV
-    when 'production', 'test' then 'https://app.suppressionlist.com'
```
Comment (cgrayson) [inline src/helper.coffee]: Wow, ha ha, oops!

File: src/query_item.coffee [hunk 2]
```diff
@@ -24,6 +25,9 @@ response = (vars, req, res) ->
     event.found_in = event.exists_in_lists
     delete event.exists_in_lists
 
+    event.added_at = time.parse _.last(_.sortBy(event.entries, 'added_at')).added_at
```
Comment (cgrayson) [inline src/query_item.coffee]: This is finding the latest timestamp, but I don't see where that behavior is tested.

**Comment by cgrayson** [comment]
Other than that question, LGTM

File: src/query_item.coffee (hunk 2 above)
Comment (bhicks) [inline src/query_item.coffee]: I updated the response of the 'should parse JSON body' test to include an entities array that matches the response from suppressionlist, and added two entries to ensure that the most recent one is returned.

I can make this a separate test if you prefer. I considered this initially, but decided to include it with the existing test since we aren't breaking down individual assertions in the happy path, and would duplicate most of the setup of this test.

Comment (cgrayson) [inline src/query_item.coffee]: Oh, my bad. I didn't compare the two timestamp strings closely enough; I thought they were identical, didn't see they're one day apart. Carry on. :smiley:
